gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=10.0.0-SNAPSHOT
+VERSION_NAME=10.1.0-SNAPSHOT
```

Comment:
is updating this line reflected in our release template?

Reply:
no, I think we should bump it when we release the v10.0.0 final. will update the release template accordingly